src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
 public class Duke {
+    // Level 6
+    //for pr creation
+    // A-trial run
+    // program to be added
+    // Find out the position of the first '/'
+    public int firstSlashTracker (String string){
+        int location = 0;
+        boolean exist = false;
```

Comment:
Should the name of the boolean be "isExist"? I also noticed the same issue in other boolean variables too.
Reference: "Boolean variables/methods should be named to sound like booleans"

Reply:
good point!